src/administration/integrations/blackfire.md

```diff
@@ -53,7 +50,29 @@ git commit -m "Enable Blackfire."
 git push
 ```
 
-### 3. Confirm it's running
+### 3. Configure your server credentials
+
+Blackfire enables to have a fine grained configuration of server credentials across branches and environments on Platform.sh.
+
+#### Configuring global server credentials
+
+Configuring server credentials on your default branch will enable you to make sure you can profile any other branch:
+
+```bash
+platform variable:create env:BLACKFIRE_SERVER_ID <insert your Server ID>
+platform variable:create env:BLACKFIRE_SERVER_TOKEN <insert your Server Token>
```

Comment:
Since the line above says this is for the "default branch", these lines should include `-e master` to ensure they go to the right place regardless of what the user has checked out at the moment.

Also, there is no "default branch".  "master branch" is the more accurate term.

Reply:
Thanks for the feedback! Fixed.